pkg/util/billing/billing.go

```diff
@@ -0,0 +1,68 @@
+package billing
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/Azure/go-autorest/autorest/azure"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+const (
+	tenantIDMSFT string = "72f988bf-86f1-41af-91ab-2d7cd011db47"
+	tenantIDAME  string = "33e01921-4d64-4f8c-a055-5bdaffd5e33d"
+)
+
+// IsSubscriptionRegisteredToE2E returns true if the subscription is having the
+// "Microsoft.ContainerService/SaveAROTestConfig" feature registered
+func IsSubscriptionRegisteredToE2E(sub *api.SubscriptionProperties) bool {
+	if sub.TenantID == tenantIDMSFT || sub.TenantID == tenantIDAME {
+		for _, feature := range sub.RegisteredFeatures {
+			if feature.Name == api.SubscriptionFeatureE2E && feature.State == "Registered" {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+// CreateOrUpdateE2Eblob create a copy of the billing document into the e2e storage account.
+// This is used later on for E2E Billing
+func CreateOrUpdateE2Eblob(ctx context.Context, env env.Interface, doc api.BillingDocument, resource azure.Resource) error {
+	key, err := env.E2EStorageAccountKey(ctx)
+	if err != nil {
+		return err
+	}
+	client, err := azstorage.NewBasicClient(env.E2EStorageAccountName(), key)
+	if err != nil {
+		return err
+	}
```

Comment:
ideally there would be a struct here containing the client, and CreateOrUpdateE2Eblob would be a method on that struct, in order to facilitate any unit testing.